db/range_del_aggregator.cc

```diff
@@ -285,7 +321,10 @@ class CollapsedRangeDelMap : public RangeDelMap {
       } else {
         // Insert a new transition at the new tombstone's start point, or raise
         // the existing transition at that point to the new tombstone's seqno.
-        rep_[t.start_key_] = t.seq_;  // operator[] will overwrite existing entry
+        end_seq = prev_range_seq();
+        CollapsedMapBoundary boundary(std::max(t.seq_, prev_point_seq()),
```

Comment:
In this case:

```
2:    c---
1: A------
```

I'm not sure `prev_point_seq()` should affect the new tombstone's point seqnum. Should it?